Speaker bios

nickmalleson opened the pull request:
Hi @urbanist-ai , I have added a couple of pictures for Linda and Sarah (keynotes) and edited the text of the Keynotes.html page a little. Please could you check that you're happy with how I've styled the images? I'm not sure whether they will behave nicely if you resise the page etc.

urbanist-ai commented:
Hi Nick,

Sure, I will do that.

Regards,
Mohamed

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Nick Malleson ***@***.***>
Sent: Tuesday, December 20, 2022 10:45:35 AM
To: urbanist-ai/urbanist-ai.github.io ***@***.***>
Cc: Ibrahim, Mohamed ***@***.***>; Mention ***@***.***>
Subject: [urbanist-ai/urbanist-ai.github.io] Speaker bios (PR #1)


⚠ Caution: External sender


Hi @urbanist-ai<https://github.com/urbanist-ai> , I have added a couple of pictures for Linda and Sarah (keynotes) and edited the text of the Keynotes.html page a little. Please could you check that you're happy with how I've styled the images? I'm not sure whether they will behave nicely if you resise the page etc.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1

Commit Summary

  *   205440d<https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1/commits/205440dec86bbb9536717358b65340efb736905e> Update README.md
  *   bf50f52<https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1/commits/bf50f52a27b4cd7f24c939c1e861c0f9a071fb55> added keynote pictures and updated bios

File Changes

(5 files<https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1/files>)

  *   M README.md<https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5> (5)
  *   A materials/keynote_headshots/LindaSee.jpg<https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1/files#diff-42479a6c79c94d43297fd3accff24075d207706cafc80c743a15cddb31fd7472> (0)
  *   A materials/keynote_headshots/headshot_sarah2022.jpg<https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1/files#diff-43c94e526fcd32f64b93c78952f5c7dc08aeee78ec79cb99406653d0d3c68b59> (0)
  *   A pages/.Keynote.html.swp<https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1/files#diff-d028c94de8b7bdcb01b307f01ccc4815028dccb0769d3da037d73804dadd058f> (0)
  *   M pages/Keynote.html<https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1/files#diff-be7edc83ff51840b70a1558daba559dc483f43a7380dc908aef02590598c42f2> (19)

Patch Links:

  *   https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1.patch
  *   https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1.diff

—
Reply to this email directly, view it on GitHub<https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AJ4FBOFYTLCN4MECCI5LZQTWOGE47ANCNFSM6AAAAAATELHQFQ>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

nickmalleson commented:
@urbanist-ai FYI I have just updated another bio (for Joao) and corrected a couple of other errors in the html (https://github.com/nickmalleson/urbanist-ai.github.io/commit/12ab4f990a361d644bfa45e7aa496eafbca41ca0)

nickmalleson commented:
@urbanist-ai sorry, I've messed up the DIV and MAIN closing tags in https://github.com/urbanist-ai/urbanist-ai.github.io/pull/1/commits/739479f208c26e8ac8782c4e009e9da6474515e7 , could you check them? I can't work out the structure now..
